app/views/ontologies/resource_content.html.haml

```diff
@@ -0,0 +1,9 @@
+= render TurboFrameComponent.new(id:"resource_content_frame_#{@format}") do
```

Comment:
do it like this one: https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/511#discussion_r1541989170